tests/test_keras_integration.py

```diff
@@ -292,13 +298,41 @@ def test_push_to_hub(self):
 
         self._api.delete_repo(name=f"{REPO_NAME}", token=self._token)
 
+    def test_push_to_hub_model_card(self):
```

Comment:
You probably want to add another test that uses `model_plot=False` and checks the image is not there